### ISSUE TITLE
black text in tables using power format pack

### ISSUE THREAD

**Issue by galantra** [krassowski/Anki-Night-Mode#78]
#### Problem description

Hi,
the issue  appears when using the tables provided by power format pack.
The text inside the tables is black when it should be white. Outside, it is white, as it is supposed to be. When I turn Night Mode off, it appears as black (on white), just like the card is styled.
See here:
![grafik](https://user-images.githubusercontent.com/52420923/61724994-48734600-ad6f-11e9-9502-1a7abcac78c2.png)
![grafik](https://user-images.githubusercontent.com/52420923/61725000-4b6e3680-ad6f-11e9-8f6a-bb6440802b2d.png)

*

#### Checklist

*Please put x inside brackets if following sentences apply to you:*

 - [X] I verified that I use the lastest add-on version,
 - [X] I verified that I use the lastest Anki version,
 - [X] I tried to disable other add-ons to find the culprit,
 - [X] I checked if anyone reported this problem before, as well as Known issues section on add-ons page,
 - [X] My issue is solved when holding shift while starting Anki.
 
#### Information about your Anki Setup

<details>

*Please open Anki, go to Help → About, look-up two lines, starting with `Version:` and `Qt` and copy-paste them below:*

```
Version: 2.0.51
Qt: 4.8.4 PyQt 4.10
```

*Please got to Night Mode → About now and copy-paste the first line from the pop-up window below:*

```
Night Mode 1.2.3
```

*Please fill in your OS details (Linux/Mac/Windows):*

```
OS: Windows 10
```
 
 
*If you have a compatibility problem with a specific add-on or add-ons, please paste the add-on page links below:*
 - [Power Format Pack: Markdown, code blocks, lists, tables, syntax highlight & more](https://ankiweb.net/shared/info/162313389)
</details>

#### Backtrace (if any)

<details>

*If you had an error which resulted in producing exception with a backtrace, please paste the backtrace below:*

```python"
[paste your backtrace in here if you got any; otherwise feel free to remove this part]
```

</details>

**Comment by krassowski** [comment]
Thank you for the detailed report. Would you be able to try again with Anki 2.1 and Night Mode 2.2.4, please? I understand that you may prefer to stick with 2.0, but I cannot volunteer to support the (very) old version any longer.

**Comment by galantra** [comment]
I will cannot test it right now but I'll come back to it in a few days.

**Comment by galantra** [comment]
Your prognosis was right, the problem is resolved in anki 2.1 and the newest addon version.

![grafik](https://user-images.githubusercontent.com/52420923/62653947-09441800-b95f-11e9-9ef7-1f0525da1b1b.png)

**Comment by krassowski** [comment]
Thanks for getting back and letting me know!
